### PR TITLE
chore: Add CODE_OF_CONDUCT

### PR DESCRIPTION
**Pull request by JabobKrauskopf**

### CODE REVIEW

File: .github/CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
+# Contributor Covenant Code of Conduct
+
+## Our Pledge
+
+We as members, contributors and leaders pledge to make participation in our
```
Comment (NicolasSiegl) [inline .github/CODE_OF_CONDUCT.md:5]:
```suggestion
We as members, contributors, and leaders pledge to make participation in our
```

File: .github/CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+
+Examples of unacceptable behavior include:
+
+- The use of sexualized language or imagery, and sexual attention or advances of
+  any kind
+- Trolling, insulting or derogatory comments, and personal or political attacks
+- Public or private harassment
+- Publishing others' private information, such as a physical or email address,
+  without their explicit permission
+- Other conduct which could reasonably be considered inappropriate in a
```
Comment (NicolasSiegl) [inline .github/CODE_OF_CONDUCT.md:36]:
```suggestion
- Other conduct that could reasonably be considered inappropriate in a
```

File: .github/CODE_OF_CONDUCT.md [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+## Enforcement Responsibilities
+
+Community leaders are responsible for clarifying and enforcing our standards of
+acceptable behavior and will take appropriate and fair corrective action in
+response to any behavior that they deem inappropriate, threatening, offensive,
+or harmful.
+
+Community leaders have the right and responsibility to remove, edit, or reject
+comments, commits, code, wiki edits, issues, and other contributions that are
+not aligned to this Code of Conduct, and will communicate reasons for moderation
```
Comment (NicolasSiegl) [inline .github/CODE_OF_CONDUCT.md:48]:
```suggestion
not aligned to this Code of Conduct. They will communicate reasons for moderation
```

**Review by NicolasSiegl — CHANGES_REQUESTED** [review]
Just some minor improvements

**Review by OFranke — APPROVED** [review]

**Review by NicolasSiegl — APPROVED** [review]
Just some minor improvements

**Pull request merged**